src/style/color-config/index.js

```diff
@@ -1,13 +1,16 @@
 // Base colors
 export default {
   brilliantGreen: '00DC00',
-  gold: 'FFB500',
   errorRed: 'C7384F',
+  gold: 'FFB500',
   genericGreen: '009900',
+  infoBlue: '0077C8',
   productGreen: '00A376',
   productBlue: '0077C8',
   navyBlue: '004B87',
   amethyst: '582C83',
   plum: '8246AF',
-  slate: '003349'
+  slate: '003349',
+  successGreen: '00B000',
+  warningOrange: 'E96400'
```

Comment:
These base colours are used to create functions which build the palette that we use in our theming library and should not be changed or renamed.